BeeSafe/beesafe-services
# connect flask to prod mysql

**What did i even do dude**
I connected our flask server to the production MYSQL db
I replaced MYSQL_HOST with the db's public IP

**How to Test**
I created a commented route that creates an example table inside the BeeSafe database.
Uncomment it and run flask to see if it creates an "example"table
Ok goodluck

**NOTE**
ONLY WHITELISTED IPs CAN ACCESS THE DB (everyone should already be whitelisted)
